pkg/config/config.go

```diff
@@ -38,36 +37,58 @@ type Set struct {
 	Configs     []Config
 }
 
+type Sysctler func(name string, params ...string) (string, error)
```

Comment:
exposed symbols should have comments, but it could be added later.

Reply:
Yeah, I will add comments later.